### PR TITLE
MF-488 - Remove Thing type (app or device)

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -106,4 +104,4 @@ mainflux-http   | {"level":"info","message":"Method publish took 336.685µs to c
 mainflux-normalizer | {"level":"info","message":"Method normalize took 108.126µs to complete without errors.","ts":"2019-01-08T22:19:30.149500543Z"}
 ```
 
-This proves that messages have been well send through the system, via protocol adapter (`mainflux-http`) and `normalizer` service which corectly parsed messages.
\ No newline at end of file
+This proves that messages have been well send through the system, via protocol adapter (`mainflux-http`) and `normalizer` service which corectly parsed messages.
```
Comment (anovakovic01) [inline docs/getting-started.md]: `correctly`

File: things/api/http/endpoint_test.go [hunk 2]
```diff
@@ -35,7 +35,7 @@ const (
 )
 
 var (
-	thing   = things.Thing{Type: "app", Name: "test_app", Metadata: map[string]interface{}{"test": "data"}}
+	thing   = things.Thing{ Name: "test_app", Metadata: map[string]interface{}{"test": "data"}}
```
Comment (anovakovic01) [inline things/api/http/endpoint_test.go]: Remove extra space in front of `Name:`.

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

**Review by anovakovic01 — COMMENTED** [review]
@manuio You'll have to remove `type` from UI too. :)

**Comment by manuio** [comment]
@anovakovic01 sure :) I'm starting right now

File: docs/getting-started.md (hunk 1 above)
Comment (manuio) [reply docs/getting-started.md]: thanks

File: things/api/http/endpoint_test.go (hunk 2 above)
Comment (manuio) [reply things/api/http/endpoint_test.go]: thanks

**Review by drasko — DISMISSED** [review]
LGTM

File: sdk/go/things_test.go [hunk 3]
```diff
@@ -87,6 +87,13 @@ func TestCreateThing(t *testing.T) {
 			err:      nil,
 			location: "1",
 		},
+		{
+			desc:     "create new epmty thing",
```
Comment (dborovcanin) [inline sdk/go/things_test.go]: Typo: `epmty`.

**Review by dborovcanin — COMMENTED** [review]
Please check these files as well:
https://github.com/mainflux/mainflux/blob/master/things/README.md#things (remove `(i.e. devices & applications)` part)
https://github.com/mainflux/mainflux/blob/master/docs/dev-guide.md#thing-create-event and https://github.com/mainflux/mainflux/blob/master/docs/dev-guide.md#thing-create-event (remove `type` field from event examples)

Great job, @manuio.

**Review by drasko — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Pull request merged**